doc/alertmanager-mixin/config.libsonnet

```diff
@@ -12,13 +13,25 @@
     // to keep for resulting cluster-level alerts.
     alertmanagerClusterLabels: 'job',
 
-    // alertmanagerName is inserted into annotations to name the Alertmanager
-    // instance affected by the alert.
-    alertmanagerName: '{{$labels.instance}}',
+    // alertmanagerQuerySelector is a matcher selector built from 'alertmanagerClusterLabels'.
+    // It is used to filter queries on grafana dashboards
+    alertmanagerQuerySelector: std.join(',', ['%s="$%s"' % [label, label] for label in std.split(c.alertmanagerClusterLabels, ',')]),
```

Comment:
Does this have to be in `_config`? I would think that this programmatic construction should always work.

Reply:
Yeah, good point. After your comment, I think it will be better as a local variable in the dashboard